target/linux/ramips/image/mt7621.mk

```diff
@@ -709,6 +709,24 @@ define Device/jcg_jhr-ac876m
 endef
 TARGET_DEVICES += jcg_jhr-ac876m
 
+define Device/jcg_q20
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  UBINIZE_OPTS := -E 5
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 32768k
```

Comment:
That's obviously wrong.

Reply:
I wrote it according to the HiWiFi HC5962, it seems that the firmware size is limited by the total size of the kernel and ubiconcat0 partition.